README.md

```diff
@@ -26,7 +26,10 @@ check out and contribute to this repository.
 Important links:
 
 - [Workshops](workshops/README.md) :: Workshops for use in clubs
-- https://slack.hackclub.io :: Join our [Slack](https://slack.com/) community
+- https://slack.hackclub.io :: Join
+  [our Slack](https://starthackclub.slack.com/) community
```

Comment:
Err, was there a reason why you linked `our Slack`? The `https://slack.hackclub.io` link is already there.